### ISSUE TITLE
static file hosting on site turned off due to error

### ISSUE THREAD

**Issue by talumbau** [OpenSourcePolicyCenter/PolicyBrain#354]
In #353 we reverted to locally hosting static files, instead of taking css and JS files, gzipping them,  and then hosting them on an S3 bucket for better load times. This was done because of an unexpected error when trying to collect the static files and upload them to the S3 bucket. When executing `python manage.py collectstatic`, the following error occured:

```
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 168, in handle
    collected = self.collect()
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 107, in collect
    handler(path, prefixed_path, storage)
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 305, in copy_file
    if not self.delete_file(path, prefixed_path, source_storage):
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 218, in delete_file
    if self.storage.exists(prefixed_path):
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/storages/backends/s3boto.py", line 452, in exists
    k = self.bucket.new_key(self._encode_name(name))
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/storages/backends/s3boto.py", line 295, in bucket
    self._bucket = self._get_or_create_bucket(self.bucket_name)
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/storages/backends/s3boto.py", line 328, in _get_or_create_bucket
    return self.connection.get_bucket(name, validate=self.auto_create_bucket)
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/storages/backends/s3boto.py", line 284, in connection
    proxy_port=self.proxy_port
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/boto/s3/connection.py", line 191, in __init__
    validate_certs=validate_certs, profile_name=profile_name)
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/boto/connection.py", line 569, in __init__
    host, config, self.provider, self._required_auth_capability())
  File "/Users/talumbau/miniconda3/envs/heroku_aei/lib/python2.7/site-packages/boto/auth.py", line 1040, in _wrapper
    if test in self.host:
TypeError: argument of type 'NoneType' is not iterable
```

It's not clear if this is a problem in `django-storages` or `s3boto` or somewhere else. But, right now, the mechanism to take the static files and have the `collectstatic` command put them in the appropriate S3 bucket is not working.

Any thoughts here? @kevincolten @brendancol

**Comment by MattHJensen** [comment]
@brittainhard, could you verify if this issue should remain open?

**Comment by hdoupe** [comment]
@GoFroggyRun can you take a look at issue #354? I think we should be able to close this. I don't think we've had any issues with css or javascript.

**Comment by GoFroggyRun** [comment]
>I don't think we've had any issues with css or javascript.

Nope. I think it's safe to close this one.

**Comment by hdoupe** [comment]
Ok, thanks @GoFroggyRun

**Comment by hdoupe** [comment]
Closed #354
